db/schema.rb

```diff
@@ -382,10 +382,10 @@
   create_table "people", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "nomis_prison_number"
-    t.string "prison_number"
-    t.string "criminal_records_office"
-    t.string "police_national_computer"
+    t.citext "nomis_prison_number"
```

Comment:
I've run migrations locally and the values are casted correctly as far as I could see. Rolling back works cleanly, too.